catalog/ui/src/app/Workshops/workshops.css

```diff
@@ -0,0 +1,91 @@
+.workshops-head.pf-c-page__main-section {
+  padding-top: 1em;
+  padding-bottom: 1em;
+  border-bottom: 1px solid var(--pf-global--palette--black-300);
+}
+
+.workshops-head.pf-c-page__main-section .pf-c-breadcrumb__item {
+  font-size: 1.2em;
+  margin-top: 0.3em;
+}
+
+.workshops-item-body.pf-c-page__main-section {
+  padding-top: 0px;
+}
+
+.workshops-item-body.pf-c-page__main-section .pf-c-tabs {
+  margin-left: -100px;
+  padding-left: 100px;
+  margin-right: -100px;
+  padding-right: 100px;
+  margin-bottom: 10pt;
```

Comment:
this isnt important for this PR, so you can ignore it for now.
but there is a reason to use pt?

Reply:
No good reason. There is also duplication between this and `ui/src/app/Services/services.css` that should get cleaned up. I think a lot of the styles need attention and rework.

Reply:
Agree, we can work on this later